socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,28 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
         for a_store in self.stores.itervalues():
-            a_store.save_raw_and_processed(
-              raw_crash,
-              dump,
-              processed_crash,
-              crash_id
+            store_name = getattr(
+                a_store, 'wrapped_object', a_store.__class__
             )
+            # self.logger.debug('writing %s to %s' % (crash_id, store_name))
```

Comment:
So, now that you've removed the debug line, the `store_name` variable is not necessarily used. 
That means you can do this line *inside* the `except Exception:` block.